### PR TITLE
 theme-obsidian2: 2.8 -> 2.9 

### PR DESCRIPTION
**Pull request by romildo**
###### Motivation for this change

Update to version [2.9](https://github.com/madmaxms/theme-obsidian-2/releases/tag/v2.9)

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/data/themes/obsidian2/default.nix [hunk 1]
```diff
@@ -2,21 +2,17 @@
 
 stdenv.mkDerivation rec {
   name = "theme-obsidian2-${version}";
```
Comment (peterhoeg) [inline pkgs/data/themes/obsidian2/default.nix]: Please change this to ```pname``` and drop the ```-${version}``` part.

File: pkgs/data/themes/obsidian2/default.nix [hunk 2]
```diff
@@ -2,21 +2,17 @@
...
+    sha256 = "1m89ws2a4nms4m8187d5cxi281b66i59xa5shlp3g1r2jc4312cy";
   };
 
   propagatedUserEnvPkgs = [ gtk-engine-murrine ];
 
-  postPatch = ''
-    sed -i -e 's|Obsidian-2-Local|Obsidian-2|' Obsidian-2/index.theme
-  '';
-
   installPhase = ''
```
Comment (peterhoeg) [inline pkgs/data/themes/obsidian2/default.nix]: Please add ```runHook preInstall``` (and ```runHook postInstall``` at the end) when overriding the whole phase.

File: pkgs/top-level/all-packages.nix [hunk 3]
```diff
@@ -17108,6 +17108,8 @@ in
     gtk = res.gtk2;
   };
 
+  theme-obsidian2 = callPackage ../data/themes/obsidian2 { };
```
Comment (peterhoeg) [inline pkgs/top-level/all-packages.nix]: Any reason for the move?

**Review by peterhoeg — CHANGES_REQUESTED** [review]

File: pkgs/data/themes/obsidian2/default.nix (hunk 1 above)
Comment (romildo) [reply pkgs/data/themes/obsidian2/default.nix]: Done.

File: pkgs/data/themes/obsidian2/default.nix (hunk 2 above)
Comment (romildo) [reply pkgs/data/themes/obsidian2/default.nix]: Done.

File: pkgs/top-level/all-packages.nix (hunk 3 above)
Comment (romildo) [reply pkgs/top-level/all-packages.nix]: Yes. There is an understanding that themes are best in the `data` category than in the `misc` one. New themes have been added there, and old themes are being moved when they are updated.

**Pull request merged**